api-contract/src/main/java/uk/gov/hmcts/fees2/register/api/contract/request/RangedFeeDto.java

```diff
@@ -24,8 +24,8 @@
     private String rangeUnit;
 
     @Builder(builderMethodName = "rangedFeeDtoWith")
-    public RangedFeeDto(String code, String newCode, FeeVersionDto version, String jurisdiction1, String jurisdiction2, String service, String channel, String event, String applicantType, BigDecimal maxRange, BigDecimal minRange, String keyword) {
-        super(code, newCode, version, jurisdiction1, jurisdiction2, service, channel, event, applicantType, false, keyword);
+    public RangedFeeDto(String code, String newCode, FeeVersionDto version, String jurisdiction1, String jurisdiction2, String service, String channel, String event, String applicantType, BigDecimal maxRange, BigDecimal minRange, String keyword, String reasonForUpdate) {
+        super(code, newCode, version, jurisdiction1, jurisdiction2, service, channel, event, applicantType, false, keyword, reasonForUpdate);
```

Comment:
What is the requirement to add 'reasonForUpdate' in as attribute in FeeDTO Model ?

Reply:
This also updated but forgot to push, did that along with Encode